plugins/easyimage/plugin.js

```diff
@@ -6,6 +6,141 @@
 ( function() {
 	'use strict';
 
+	var commandsToRefresh = [ 'easyimageFull', 'easyimageSide' ];
+
+	function isSideImage( widget ) {
+		return widget.element.hasClass( 'easyimage-side' );
+	}
+
+	function isFullImage( widget ) {
+		return !isSideImage( widget );
+	}
+
+	function refreshCommands( editor, commands ) {
+		CKEDITOR.tools.array.forEach( commands, function( command ) {
+			editor.getCommand( command ).refresh( editor );
+		} );
+	}
+
+	function createCommandRefresh( check ) {
```

Comment:
How about `activeCheck` / `enabledCheck` or anything along these lines? The reason why I'm asking is `check` says almost nothing about the purpose of variable.